packages/stencil/src/components/datacom-radio/useage.md

```diff
@@ -7,5 +7,12 @@
 
   <datacom-radio label="Radio" id="radio1" value="choice3" name="choose" variant="buttons" size="small" disabled="true"> </datacom-radio>
```

Comment:
Could you rename this file to usage.md? 'Useage' is actually misspelt.

Reply:
Fixed it.